sdk/monitor/Azure.Monitor.Ingestion/tests/Samples/Sample1_HelloWorld.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Monitor.Query;
+using Azure.Monitor.Query.Models;
+
+namespace Azure.Monitor.Ingestion.Tests.Samples
+{
+    public partial class IngestionSamples: SamplesBase<IngestionClientTestEnvironment>
+    {
+        public void LogDataAndQuery()
+        {
+            #region Snippet:Azure_Monitor_Ingestion_Scenario
+            Uri dataCollectionEndpoint = new Uri("...");
+            TokenCredential credential = new DefaultAzureCredential();
+            string workspaceId = "...";
+#if SNIPPET
+#else
+            dataCollectionEndpoint = new Uri(TestEnvironment.DCREndpoint);
+            credential = TestEnvironment.ClientSecretCredential;
+#endif
+            LogsIngestionClient client = new(dataCollectionEndpoint, credential);
```

Comment:
For code snippets, let's be consistent with the approach. For example, a target-typed new expression is used here, but not on the next line.